Mlem/App/Views/Root/Tabs/Feeds/SubscriptionListView.swift

```diff
@@ -8,54 +8,193 @@
 import MlemMiddleware
 import SwiftUI
 
-private struct SubscriptionListSection: Identifiable {
-    let label: String
-    let communities: [Community2]
+struct SubscriptionListView: View {
+    @Environment(AppState.self) private var appState
+    @Environment(NavigationLayer.self) private var navigation
     
-    var id: String { label }
-}
-
-private extension SubscriptionList {
-    var visibleSections: [SubscriptionListSection] {
-        var sections: [SubscriptionListSection] = .init()
-        if !favorites.isEmpty {
-            sections.append(.init(label: "Favorites", communities: favorites))
-        }
-        for section in alphabeticSections.sorted(by: { $0.key ?? "~" < $1.key ?? "~" }) {
-            sections.append(.init(label: section.key ?? "#", communities: section.value))
+    @AppStorage("subscriptions.sort") private var sort: SubscriptionListSort = .alphabetical
+    @AppStorage("subscriptions.instanceLocation")
+    private var savedInstanceLocation: InstanceLocation = UIDevice.isPad ? .bottom : .trailing
+    
+    @State var noDetail: Bool = false
+    
+    var body: some View {
+        Group {
+            if UIDevice.isPad {
+                content
+                    .listStyle(.sidebar)
+            } else {
+                content
+                    .listStyle(.plain)
+            }
```

Comment:
Should be `MultiplatformView`; alternatively a little more concise (if less flexible in case we want more platform-specific behavior) as `content.listStyle(UIDevice.isPad ? .sidebar : .plain)`